cmake/find/Findtiny-process-library.cmake

```diff
@@ -0,0 +1,58 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
We shouldn't be using find modules for getting package details - export a CMake config file instead and remove this file. This means we'll have to fork the upstream project, at least for now.

Reply:
Yep I know. Sorry I was "lazy" because my main goal is to have [tinyrefl](https://github.com/Manu343726/tinyrefl) in hunter .... but it has so many dependencies :-(